src/common/components/clusterConfiguration/CertificateFields.css

```diff
@@ -0,0 +1,3 @@
+.ai-certificate-fields .pf-c-check__body {
```

Comment:
Could we not add the utility class `pf-u-w-100` instead?
Maybe on `CertificateInputFields` ?